### DISCUSSION TITLE
Any plan to support parallel computation for cluster points?

### DISCUSSION THREAD

**Discussion by timberUp** [freesinger/density_peak_clustering#11] [Q&A] [answered]
Computing density peak point costs too much time for big datasets.
Any good solution？

**Comment by freesinger** [comment, marked as answer]
hi @timberUp，thanks for your interesting
https://github.com/freesinger/density_peak_clustering/blob/d3ee8b991c1603b596181bd41700c1c2209a39c0/data_process.py#L61
You may check this line and change the 3rd parameter to a integer value `[1, 100)`. Smaller the parameter, less the time elapsed. Also, the precision would be reduced.
No parallel computation plan for this since this repo has been half-archived for now. You could devesigate parallism mechanism in Python.
Hope it will help~
